cfme/tests/infrastructure/test_system_image_type.py

```diff
@@ -9,41 +9,46 @@
 pytestmark = [pytest.mark.tier(3)]
 
 
-def test_system_image_type_crud():
+@pytest.fixture(scope="module")
+def collection(appliance):
```

Comment:
**REQUIRED:** This is too generic a name, I actually don't think a fixture is necessary here and would prefer to use the appliance call each time in the test, requiring the appliance fixture instead.

Reply:
@psav I would be thankful if you explain the point of using appliance call each time.